src/BenchmarkDotNet/Portability/Cpu/HardwareIntrinsics.cs

```diff
@@ -9,6 +12,83 @@ namespace BenchmarkDotNet.Portability.Cpu
 {
     internal static class HardwareIntrinsics
     {
+        internal static string GetVectorSize()
+        {
+#if NET6_0_OR_GREATER
+            if (Vector.IsHardwareAccelerated)
+                return $"VectorSize={Vector<byte>.Count * 8}";
+#endif
+            return string.Empty;
+        }
+
+        internal static string GetShortInfo()
+        {
+            if (IsX86Avx2Supported)
+                return "AVX2";
+            else if (IsX86AvxSupported)
+                return "AVX";
+            else if (IsX86Sse42Supported)
+                return "SSE4.2";
+            else if (IsX86Sse41Supported)
+                return "SSE4.1";
+            else if (IsX86Sse3Supported)
+                return "SSE3";
+            else if (IsX86Sse2Supported)
+                return "SSE2";
+            else if (IsX86SseSupported)
+                return "SSE";
+            else if (IsArmAdvSimdSupported)
+                return "AdvSIMD";
+            else if (IsArmBaseSupported)
```

Comment:
Base should likely be returned rather than empty string or it should also check for x86base